frontend/packages/dev-console/src/components/topology/TopologyDataController.tsx

```diff
@@ -32,62 +33,44 @@ export interface ControllerProps {
   loadError?: any;
   resources?: TopologyDataResources;
   render(RenderProps): React.ReactElement;
-  application: string;
-  cheURL: string;
+  namespace: string;
   serviceBinding: boolean;
-  topologyFilters: TopologyFilters;
   trafficData?: TrafficData;
   helmResourcesMap?: HelmReleaseResourcesMap;
 }
 
 export interface TopologyDataControllerProps extends StateProps {
   namespace: string;
   render(RenderProps): React.ReactElement;
-  application: string;
-  knative: boolean;
-  cheURL: string;
-  serviceBinding: boolean;
 }
 
 const Controller: React.FC<ControllerProps> = ({
   render,
-  application,
-  cheURL,
   resources,
   loaded,
   loadError,
   utils,
+  namespace,
   serviceBinding,
-  topologyFilters,
   trafficData,
   helmResourcesMap,
-}) =>
-  render({
+}) => {
+  return render({
```

Comment:
I assume a debugging refactor? Not sure this change is needed, but no skin off my nose to leave it.